stripe-core/src/main/java/com/stripe/android/core/networking/AnalyticsRequest.kt

```diff
@@ -10,10 +11,12 @@ import androidx.annotation.RestrictTo
  * this table through r.stripe.com. See [AnalyticsRequestV2] for details.
  */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Serializable
 data class AnalyticsRequest(
-    val params: Map<String, *>,
-    override val headers: Map<String, String>
+    val params: Map<String, String?>,
```

Comment:
We need to convert this from `Map<String, Any?>` to `Map<String, String?>` so that we can serialize it for the `Worker` class. These parameters were being converted in `QueryStringFactory` anyway, so we’re merely pulling it forward.